app/models/server.rb

```diff
@@ -44,7 +43,7 @@ class Server < ActiveRecord::Base
   attr_protected :id, :user_id, :updated_at, :created_at, :map, :players, :maxplayers, :ping, :version
 
   validates_length_of [:name, :dns,], :in => 1..30
-  validates_length_of [:rcon, :password, :irc], :maximum => 30, :allow_blank => true
+  validates_length_of [:password, :irc], :maximum => 30, :allow_blank => true
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer the new style validations `validates :column, length: value` over `validates_length_of`.